modules/ROOT/pages/design-import-files.adoc

```diff
@@ -3,51 +3,69 @@ ifndef::env-site,env-github[]
-+
-Additional specifications that you import can be in RAML or OAS.
+====
 
-Importing API fragments::
-+
-Both API specifications and API fragments can include reusable portions that are contained in separate files. For example, suppose you want to use OAuth 2.0 for securing implementations of your API. You might have a security scheme defined in a RAML file named `oauth_2_0.raml` that explains the settings for implementations to use, as well as other info such as headers and responses. You would import `oauth_2_0.raml` into your project. Then, in your API specification, you would include the security scheme, as in this example:
-+
-....
-securitySchemes:
-  - oauth_2_0: !include oauth_2_0.raml
-....
-+
-Then, use the `securedBy` keyword to refer to the scheme, as in this example:
-+
-....
-securedBy: [ oauth_2_0 ]
-....
-+
-You can import annotation types, data types, examples of JSON response payloads, libraries, resource types, security schemes, and traits. If you import an example of a JSON response payload, you can also import the JSON schema, so that developers creating their own response payloads can refer to the schema when doing so.
+The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can import annotation types, data types, examples of JSON response payloads, libraries, resource types, security schemes, and traits.
+
+== Procedure
+
+* *In the API editor only:* To import a file or the content of a .zip file that is stored on your computer:
+. In the left pane of the editor, click the dots to the right of *Files* and select *Import*.
+. In the *Import* dialog, select *File or Zip*.
 
++
+*Result:* The file or files are listed in the left pane of the editor. If you imported a `.zip` file that contains files that are in a folder structure, the folder structure is represented in the pane.
 
+* *In the API editor only:* To import a file that is available through a URL:
 
+. In the left pane of the editor, click the dots to the right of *Files* and select *Import*.
+. In the *Import* dialog, select *URL*.
++
 [NOTE]
 ====
 
 * If you import a file that is available online by providing its URL, be sure that the file contains no `!include` statements (RAML) or `$ref` statements (OAS). The editor imports only the file that is at the URL.
 
 ====
 
++
+*Result:* The file is listed in the left pane of the editor.
 
-== Procedure
+* To import an API specification or API fragment from Exchange, together with its dependencies:
+. Open the *Import Asset from Exchange* dialog by using either of these methods:
++
+** *In the API editor only:* If your project contains only the default file that API Designer created together with the project, click the linked text that appears in the *Files* pane.
```

Comment:
I think in this case, *In the API editor only:* should not be *bold* since this styling is reserved for menus or options from the UI.

Reply:
I'll make the changes. I'm not sure that only UI text is allowed to be in bold. However, I don't think bold is necessary here and in similar places in this topic.